zebra-network/src/constants.rs

```diff
@@ -108,6 +108,15 @@ lazy_static! {
     }.expect("regex is valid");
 }
 
+/// The timeout for DNS lookups.
+///
+/// [6.1.3.3  Efficient Resource Usage] from RFC [Requirements for Internet Hosts]
```

Comment:
Just a minor tidy up - most RFCs are identified by number and title
```suggestion
/// [6.1.3.3  Efficient Resource Usage] from [RFC 1123: Requirements for Internet Hosts]
```